### ISSUE TITLE
Find sponsors

### ISSUE THREAD

**Issue by plexus** [clojurebridge-berlin/organization#44]
I can take the lead in this one, since I will handle the finances in general, but everybody can help with this. Ask people you know, tweet about it, etc.

Comment on here if you contacted a specific company so we know who is contacting whom.

You can email potential sponsors this link, this should answer the most immediate questions they have. http://clojurebridge-berlin.github.io/sponsor-us/ Feel free to tweet that link as well.

When it comes to specifics of payments, getting receipts from Travis etc, just add me in CC and I can follow up on it.

Potential sponsor. Box ticked == has been contacted.
~~strike through~~ == out
**bold** == accepted
- [x] **Pineapple Media**
- [x] **Wunderlist/Microsoft**
- [x] **Fy**
- [x] **Babbel**
- [x] **SoundCloud**
- [x] Futurice
- [x] **Ticketsolve**
- [ ] Dawanda (Malwine)
- [x] Hitfox
- [x] InnoQ
- [ ] Asquera
- [x] Github (Martin)
- [x] Thoughtworks
- [x] Cognitect
- [x] Juxt (Martin)
- [x] ~~Blacklane~~
- [x] O'Reilly
- [x] **PurelyFunctional.tv**
- [ ] Pragmatic Programmers
- [ ] Jayway
- [x] ImmoScout
- [x] Zalando
- [x] Otto
- [x] codecentric
- [x] Mixcloud
- [x] Twilio
- [x] Tape.tv
- [x] ~~Contenful~~
- [x] ~~Siili~~
- [x] ~~Adjust~~ (Martin)
- [x] **Metosin**
- [x] **Lambdawerk**
- [x] MixRadio
- [x] ableton
- [x] **bitcrowd**

EDIT:
**Amount overview:**
https://github.com/clojurebridge-berlin/organization/blob/master/confirmed_sponsors.md

**Comment by Malwine** [comment]
I can ask my contact at Cognitect and DaWanda(not so sure about them though).

**Comment by nicsnet** [comment]
@plexus @Malwine I need credentials for the clojure bridge gmail account. 

So far I contacted Ben from RPM, and he confirmed that they'll sponsor something if we go through the official email and contact the CEO with a list of potential items to sponsor.

Babbel is out of budget, but I'll hear back from them next week, if they can loosen it a bit.

**Comment by nicsnet** [comment]
Fy confirmed that they're sponsoring drinks.

**Comment by nicsnet** [comment]
Minimum sponsorship requirement for T-shirt logo placement. Sponsors are of course interested in having their logo displayed on the T-shirt. I got some feedback though, regarding our sponsorship options. One sponsor went for the lowest possible option 30 euros. Does this still qualify them for having their logo displayed? Last time i remember that price was just pocket expense for someone.

**Comment by nicsnet** [comment]
I think the sponsor us page could be updated and options provided. So that you get a T-shirt logo for minimum of 200 euros contribution (or along those lines).

**Comment by plexus** [comment]
I don't think we should take sponsors for 30 euros. That's just not worth it considering the overhead of emails, finances, paperwork. I'm sure we can find real sponsors in the 200-600 range.

I am very happy we didn't have sponsor logos on the t-shirts, this means our t-shirts don't make you feel like a walking advertisement, which in turn means people actually wear the t-shirts. If you look at conference t-shirts it's quite rare for those to include sponsors. So my vote is on keeping sponsor logos off the t-shirts.

**Comment by jellea** [comment]
maybe we should get rid of that example then...

**Comment by plexus** [comment]
> We also list sponsors **by name** on our popular t-shirts

emphasis added :)

**Comment by Malwine** [comment]
:+1: to all the things discussed
(sorry am in a hurry)

**Comment by nicsnet** [comment]
I didn't really mean sponsor logo, i meant sponsor name. Sorry for the confusion :confused: 

Anyway i agree, taking on sponsors for less than 200 euros isn't really worth the hassle.

**Comment by thatbettina** [comment]
@Malwine @nicsnet @plexus Have any of you got an email template of sorts for contacting the sponsors? I have what I used last year, but I thought that it might be nice to add what we did for the first workshop, how many people came, the fact that there's been 2-3? project groups that came out of it, who WE are.

Also, @plexus I noticed that the "Sponsor Us" information doesn't refer to a specific workshop but rather to sponsoring us as a group. I think for non-tech people (or even some tech people) it might not be obvious what we do and what the workshops are, especially if they don't think to look at http://clojurebridge-berlin.github.io/  What's your experience of sending sponsors to this page?

**Comment by plexus** [comment]
I'll work on some better copy to approach sponsors with. I originally didn't really want to formulate specific "sponsor packages" because really people should sponsor us because they care, not because it's a good marketing opportunity. But the experience with RPM is that we clearly need to set some boundaries and expectations.

**Comment by Malwine** [comment]
@thatbettina I don't have an email template. But if you or anybody else has one dont hesitate to put it in a markdown file :smile:

**Comment by nicsnet** [comment]
Blacklane just got back to me, they're out.

**Comment by plexus** [comment]
Contacted Cognitect, Tape.tv, Twilio, Contentful. Contentful is out, the rest I don't know yet.

So far I've told people that a typical sponsorship is in the 400 to 600 euro range, although we accept anything from 200 euro up.

**Comment by jellea** [comment]
Got back from Zalando that they are interested, but due to planning will answer mid-january. Responded that we can't wait that long because material (mainly tshirts) need to be printed

**Comment by plexus** [comment]
@thatbettina and the rest, I don't really have an email template because how I phrase things depends a lot on the person or the company, but I put the last couple emails I set in a file so you can take ideas from there.

https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/sponsor-mails.md

Suggestions welcome. For now I'm mostly leaving it up to the companies to make a proposal, just giving a suggested range. It would probably be easier to state specific tiers up front, that's something we should work out for next time.

e.g. 
- gold sponsor, €1000 euro, more prominently on site, slides, etc, plus gets to do a lightning talk at the event
- silver sponsor €600 euro, ...
- bronze sponsor €300 euro, ....

**Comment by Malwine** [comment]
I feel these ranges are very high for a workshop in our size. :fearful:

**Comment by plexus** [comment]
It's just an example, and to be discussed and fleshed out after this workshop, but I think it's not that unreasonable. Last time we spent 2300+ euro. If we want to get that with five sponsors then that's 460 each. For a profitable company these aren't massive amounts.

Last time we got 157.24, 220, 400, 400, 500, and 655.21, so an average of 388, or a mean of 400.

Having one expensive upper tier is good psychologically because people more easily go for the second most expensive option then for the most expensive option. It's the same when selling wine for instance. If you have bottles of 5 euro, 8 euro and 12 euro, then most people will take 5 or 8, but if you put a 20 euro bottle next ot it more people get the 12.

**Comment by Malwine** [comment]
I see your point for the sponsoring.

For some time now I wanted to start a discussion (maybe in ClojureVerse, **not here**) on the finance in general. Maybe we should rethink how much money we spend and how much we need. What the key costs are and what extra "cool" stuff is.

**Comment by martinklepsch** [comment]
If someone knows the numbers for the Fy/Wunderlist/RPM sponsorships it would be good to put them next to the names so we have a rough idea how much is covered already. RPM was 200 I believe?

**Comment by plexus** [comment]
see https://github.com/clojurebridge-berlin/organization/blob/master/confirmed_sponsors.md

**Comment by thatbettina** [comment]
@Malwine I'm with you on discussing the finances in a private space. However I'm not sure if I'm on ClojureVerse??

**Comment by thatbettina** [comment]
I will follow up with ImmoScout tomorrow regarding sponsoring. We are totally busy with the end of year planning for next year, however, with a huge meeting on Thursday, so I may not get an answer until Friday/early next week.

**Comment by vsmart** [comment]
Digging through contacts from clojureX to see who would be interested. @plexus - I can't edit the original list as I'm not in the organization. Could you add `[x] metosin`? 

ETA: thanks @jellea ^-^ I have edit power now :dancers:

**Comment by jellea** [comment]
@vsmart I've added you some time ago. Look in your github notifications for the invite.

**Comment by vsmart** [comment]
Good news! I heard back from lambdawerk and Metosin and they will both help with sponsoring. (200 from Metosin - not sure about the amount from LW yet).

**Comment by nicsnet** [comment]
Babbel confirmed \o/ .They are contributing 200 Euros.

**Comment by martinklepsch** [comment]
There is a cross for GitHub — did someone contact them already? I just sent of the form on http://community.github.com/ after not hearing back via email directly. 

**EDIT:** Probably good to add names behind sponsors names so we know who is in touch/responsible.

**Comment by jellea** [comment]
@vsmart @nicsnet great! what is the current total?

**Comment by martinklepsch** [comment]
https://github.com/clojurebridge-berlin/organization/blob/master/confirmed_sponsors.md + metosin w/ 200€ I think

**Comment by tat-ableton** [comment]
Hi there! I know a Community Manager at O'Reilly and I reached out to ask him if they would be interested and or to give me a better address to ask about it. I will also contact Hitfox and of course Ableton :). I will let you know if I can think of any other companies.

**Comment by TamaraAtanasoska** [comment]
Hi there! I know a Community Manager at O'Reilly and I reached out to ask him if they would be interested and or to give me a better address to ask about it. I will also contact Hitfox and of course Ableton :). I will let you know if I can think of any other companies.

**Comment by plexus** [comment]
Ticketsolve has confirmed, they put in 360 for the catering of the installfest.

**Comment by Malwine** [comment]
@TamaraAtanasoska @plexus :100: :heart:

**Comment by vsmart** [comment]
I've spoken to bodo from Bitcrowd and they will contribute 500 Eur for sponsoring catering of the workshop! :tada: 
Glad to see that our finances are looking pretty good now.

**Comment by plexus** [comment]
Oh that's is awesome news! Great way to roll into Christmas :) :confetti_ball:

**Comment by plexus** [comment]
Also, attendees can get access to LispCast, see this Twitter convo. I'll send Eric Normand an email about this, we'll need to get discount codes to hand out https://twitter.com/ericnormand/status/677267079528075266

**Comment by martinklepsch** [comment]
I contacted GitHub for sponsorship again (17.12.) via http://community.github.com/ after they haven't replied to plain emails but unfortunately haven't heard back yet. Pinged them twice since new years but I guess we can't rely on them this time.

**Comment by plexus** [comment]
Soundcloud just got back to us, they want to sponsor the installfest food, but that's already covered by Bitcrowd. I'm checking if we can work something else out.

**Comment by TamaraAtanasoska** [comment]
An update: Ableton passed unfortunately, and got no reply from my contacts at O'Reilly and Hitfox, maybe someone can tap into them from different channels? I contacted Nokia HERE since I heard they also used Clojure a bit inside, waiting on that one.

**Comment by plexus** [comment]
**Soundcloud** has confirmed, waiting on their finance people now to sort the payment.

**Metosin**, **bitcrowd**, **LambdaWerk**, and earlier **Red Pineapple Media** have all paid, I've forwarded their details to Travis Foundation.

**Babbel** is refusing to pay without some document up front, which is not something we do. We'll have to see how this plays out, if this can work for a dozen other companies I don't see why we need to do things differently for them. 

The alternatives are to either have them pay for something directly, or to keep invoices/receipts for various expenses and have them refunded. I'm not a fan of the last option, because it creates a lot of extra work for us. If we find a good match for an expense they can order themselves that might work though? Maybe the notebooks, or stickers?

**Ticketsolve** also still needs to pay, I'll follow up with them.

**Fy** is sponsoring the drinks, how did that work last time? I know Dawanda paid for them last time but I have no idea how it was organized. Who ordered the drinks, and from where? Was there an invoice? When and to whom was the payment made?

And finally some more good news, still waiting for final confirmation but **ImmoScout** is likely also in as a sponsor!

**Comment by Malwine** [comment]
@plexus With DaWanda we ordered the drinks at our supplier and managed everything around it.
If I ask DaWanda for sponsoring - that might be the thing they would like to do again. So my question is could Fy sponsor the money? Or are they set on the drinks?

**Comment by plexus** [comment]
Pinging @pesterhazy

**Comment by jellea** [comment]
Ok, asked Fy and they are just going to give us money (250 euro) so we can take care of the drinks ourselves. Can someone send payment details to Fy?

**Comment by plexus** [comment]
Sure, can you send me a contact email address?

**Comment by Malwine** [comment]
I am currently asking if DaWanda can care for the drinks.

**Comment by Malwine** [comment]
**Update**: DaWanda pays and takes care of the drinks (delivery, payment)

**Comment by plexus** [comment]
Good news: Zalando is pitching in €500 without needing extra perks.

The bad news: these companies still haven't (to my knowledge) paid
- Fy 250
- Babbel 200
- Ticketsolve 360
- Soundcloud 360
- Zalando 500 (well ok let's give them a few days)

That means €1150 has been paid, and we're still waiting for €1670. I sent a follow-up email to Fy, Ticketsolve, and Soundcloud to see what's up.

It seems this time around we need to follow up and poke the sponsors much more than last time. I'm not sure if that's because of a shorter time frame, because we're doing almost everything through Travis now (vs a number of direct invoices last time), or if it just feels that way because we have more individual sponsors.

**Comment by plexus** [comment]
With that I would also say we're officially done accepting sponsors for this edition. But, if you get any questions from companies, tell them we're accepting sponsoring for the next edition :)

**Comment by plexus** [comment]
And it's our lucky day, Cognitect just got back to me offering "something in the 500 euro range". 

since it's so late and since our costs are well covered already I will ask if we can take them up on that offer for the next workshop. 

They do ask if it's possible for them to pay directly for something by credit card, so we'll have to figure out what that can be.

**Comment by Malwine** [comment]
WOW :sparkles: :money_with_wings: :moneybag: :heart_eyes: :heart_decoration: 
such money
many future workshops convered
amaze people

Well, but really! That's awesome!

**Comment by plexus** [comment]
Reopening as this isn't really wrapped up yet, we're still waiting for payment from three companies, for a total of 1060 euro.

| company | amount | last contacted |
| --- | --- | --- |
| Babbel | 200 | 15 January |
| Soundcloud | 360 | 18 January |
| Zalando | 500 | 20 January |

I'll give it a few more days then send them all another reminder. It's hard to express how tired I am of chasing these people.

The rest of the sponsoring is mostly wrapped up, I received receipts from Travis Foundation today and forwarded them. They were in German, for Metosin and Ticketsolve Anika will prepare receipts in English, so I'm still waiting for those.

**Comment by thatbettina** [comment]
@plexus Can we close this or do you need help with the invoices?

**Comment by plexus** [comment]
Not invoices, donation receipts :P 

It's fine. Babbel and Soundcloud have paid and gotten their receipt. Zalando aked for a specific "cover letter" which I wrote, so I'm assuming that's fine. If anyone feels like further following up with them the emails are all in our gmail. Just a quick "did the cover letter suffice, has it been paid, if so when?"

I just realized I never got the English receipts for Metosin and Ticketsolve. I'll check with Anika, these might have been forgotten while taking care of other things.

**Comment by plexus** [comment]
Metosin and Ticketsolve have gotten their receipts now. That means it's all wrapped up except for Zalando. They haven't responded since I sent that "cover letter" they asked for.

**Comment by vsmart** [comment]
Thanks for the update! I hope Zalando will get it together soon.

**Comment by plexus** [comment]
A new person from Zalando helpfully got back to me saying that it's simple, they just need an invoice. (snip sarcastic rant)

I explained again in detail the thing with the donation receipts. If that doesn't do it I suggest we give up on this money and instead find them something they can pay directly for the next workshop. They agreed to sponsor €500 so that could be a good food budget.

**Comment by plexus** [comment]
Zalando paid up, I forwarded the details to Travis. Now the only thing left to do is forward them their receipt once we get it, but I'm going to optimistically close this already :)

**Comment by vsmart** [comment]
Finally. Thank you for following up. :)
